### DISCUSSION TITLE
Post scan callback

### DISCUSSION THREAD

**Discussion by Rahul-krishnamoorthy1** [zpao/qrcode.react#194] [Q&A] [answered]
Hello Everyone, 

I have got a quick question. 

Does this module having any props related to post scanning the barcode ? (e.g) - onScan callback.
If not any scope of adding it in near future ?

**Comment by zpao** [comment, marked as answer]
AFAIK there is no way to know if the qrcode has been scanned, unless the value is a url and you have some communication with the server but that falls squarely in the app level to handle.
